nav2_motion_primitives/include/nav2_motion_primitives/back_up.hpp

```diff
@@ -40,7 +40,7 @@ class BackUp : public MotionPrimitive<nav2_tasks::BackUpCommand, nav2_tasks::Bac
   double linear_acc_lim_;
 
   nav_msgs::msg::Odometry::SharedPtr initial_pose_;
-  double command_x_;
+  double command_x_2_;
```

Comment:
Can we add a comment here explaining this is command x squared? I think I like this notation, but, not being familiar with it, it threw me for a loop till later in the diff.

Reply:
I prefer `command_x_sq_` over `command_x_2_`

Reply:
`sq` is fine, adding `2` at the end is usually just what I do by default. Will change.